bundles/org.openhab.binding.linky/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="linky" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- Linky Thing -->
+	<thing-type id="linky">
+		<label>Linky</label>
+		<description>
+		Provides your energy consumption data. 
+		In order to receive the data, you must activate your account at https://espace-client-particuliers.enedis.fr/web/espace-particuliers/compteur-linky.
+		</description>
+		
+		<channel-groups>
+			<channel-group typeId="hourly" id="hourly" />
+			<channel-group typeId="daily" id="daily" />
+			<channel-group typeId="monthly" id="monthly" />
```

Comment:
I think you should add a group for weekly data.

Reply:
Yes, I let this in a first time because I had no view over "hourly" data. Now I have. It's a bit frustrating because hourly data are yesterdays hour data. Not today hours.